dist/react-select.es.js

```diff
@@ -1315,7 +1336,7 @@ var Select$1 = function (_React$Component) {
 			} else {
 				this.setState({
 					inputValue: this.handleInputValueChange(''),
-					isOpen: !this.props.closeOnSelect,
+					isOpen: this.props.type === 'unique' ? true : !this.props.closeOnSelect,
```

Comment:
You can simplify this to  `this.props.type === 'unique' || !this.props.closeOnSelect`